ex11/part4/src/index.js

```diff
@@ -0,0 +1,25 @@
+import React, {useState} from "react";
+import ReactDOM from "react-dom";
+import Cb from "./colorbox";
+import Cd from "./colordiv";
+
+const App = () => {
+    const [color, setColor] = useState("red");
+    function handleChangeText(e) {
+        const newValue = e.target.value;
+        setColor(newValue);
+    }
+
+    return (
+        <div>
+            <h1>ex 11 part 4</h1>
+            <Cb toUpdate={handleChangeText} defaultColor={color}/>
```

Comment:
Name...:)

Also, all the 3 components you have can be 1. No reason whatsoever to split them.